src/utilities.ts

```diff
@@ -1,4 +1,9 @@
-import { FirestoreSemesterType } from './user-data';
+import type { FirestoreSemesterType } from './user-data';
+
+export function checkNotNull<T>(value: T | null | undefined): T {
+  if (value == null) throw new Error();
+  return value;
+}
```

Comment:
I introduced this function so that all the not-null assertion is more explicit, and to make linter happier.